source/compiler-core/slang-misc-diagnostic-defs.h

```diff
@@ -24,7 +24,11 @@
 
 DIAGNOSTIC(-1, Note, seeTokenPasteLocation, "see token pasted location")
 
-DIAGNOSTIC(21, Error, expectedArgumentForOption, "expected an argument for command-line option '$0'")
+DIAGNOSTIC(100000, Error, downstreamNameNotKnown, "downstream tool name not known")
```

Comment:
Could this be something like `unknown downstream tool '$0'` so that it includes which tool is unknown?